content/en/docs/reference/command-line-tools-reference/feature-gates/VolumeAttributesClass.md

```diff
@@ -17,6 +17,10 @@ stages:
   - stage: stable
     defaultValue: true
     fromVersion: "1.34"
+  - stage: stable
+    defaultValue: true
+    locked: true
```

Comment:
<img width="1041" height="563" alt="Image" src="https://github.com/user-attachments/assets/b6f28eea-7e21-4c2f-b6f6-3ea2249de3ea" />

The locked field is not used in the `feature-gate-table.html` file. Do we forget to add it or it is unsupported?

<img width="934" height="165" alt="Image" src="https://github.com/user-attachments/assets/b72f21c7-26d8-4505-86e8-97135dccbf02" />

`VolumeAttributesClass` feature gate was marked as GA in 1.34, but the locked field was added in 1.36. How to handle this case?